beacon-chain/powchain/service.go

```diff
@@ -295,6 +302,28 @@ func (w *Web3Service) ProcessChainStartLog(depositLog gethTypes.Log) {
 	w.chainStartFeed.Send(chainStartTime)
 }
 
+func (w *Web3Service) runDelayTimer(done <-chan struct{}) {
+	timer := time.NewTimer(time.Duration(w.chainStartDelay) * time.Second)
+
+	for {
+		select {
+		case <-done:
+			log.Debug("ETH1.0 chain service context closed, exiting goroutine")
+			timer.Stop()
+			return
+		case currentTime := <-timer.C:
+
+			w.chainStarted = true
+			log.WithFields(logrus.Fields{
+				"ChainStartTime": currentTime.Unix(),
+			}).Info("Minimum Number of Validators Reached for beacon-chain to start")
```

Comment:
why capitalized `Number Validators Reached`? Shouldn't it be standard sentence format?

Reply:
Alright done